style.css

```diff
@@ -106,3 +110,202 @@ article {
 .social-media {
   margin-left: 26px;
 }
+
+.work-section {
+  margin: 420px 0 150px;
+  display: grid;
+  row-gap: 100px;
+  padding: 0 5%;
+  justify-items: center;
+}
+
+.work-card {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
```

Comment:
- [ ] As demanded b project requirements, kindly use CSS flexbox to layout contents of individual cards.
![image](https://github.com/Afsheen96/Portfolio/assets/94551484/7595cfdc-8822-465e-ad40-bc2f74d323dd)